lib/Runtime/Library/JavascriptDate.cpp

```diff
@@ -267,11 +272,6 @@ namespace Js
                 //anything else should throw a type error
             }
         }
-        else if (args.Info.Count == 1)
-        {
-            //7.1.1 ToPrimitive Note: Date objects treat no hint as if the hint were String.
-            return JavascriptConversion::OrdinaryToPrimitive(args[0], JavascriptHint::HintString/*tryFirst*/, scriptContext);
-        }
 
         JavascriptError::ThrowTypeError(scriptContext, JSERR_FunctionArgument_Invalid, _u("Date[Symbol.toPrimitive]"));
```

Comment:
Sure - updated.

---
In reply to: [71950085](https://github.com/Microsoft/ChakraCore/pull/1319#discussion_r71950085) [](ancestors = 71950085)